donat/_donations.dm

```diff
@@ -0,0 +1,167 @@
+/datum/donate_info
```

Comment:
You honest to god put micro transactions into your downstream? Why?

Reply:
Brother, what?

Reply:
> You honest to god put micro transactions into your downstream? Why?

I accidentally made this pull request to the repository

Reply:
That doesn't answer the question.